src/main/java/core/basesyntax/PeoplesSalary.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+
+public class PeoplesSalary {
```

Comment:
```suggestion
public class EmployeeWorkingDay {
```
and add method `getSalary` to this class. This method should `return salaryPerHour * numberOfHours`